config/main.py

```diff
@@ -1579,5 +1589,134 @@ def del_ntp_server(ctx, ntp_ip_address):
     except SystemExit as e:
         ctx.fail("Restart service ntp-config failed with error {}".format(e))
 
+g_ext_cfg = {}
+
+@config.group('export', invoke_without_command=False)
```

Comment:
This 'export' group does not appear to be relevant to the 'system ready' changes as stated in the PR title and description. Can these be removed and a separate PR created for them?